### ISSUE TITLE
Provide a validation set when training neural networks

### ISSUE THREAD

**Issue by marfox** [Wikidata/soweego#281]
By default, `Keras` **doesn't hold out** a validation set from the training, so we assume it directly computes the error on the training.
